### PR TITLE
fix: [CDS-37138]: fixed defaultTab issue and css fix

### PR DESCRIPTION
**Pull request by Sagar-Rajput7**
##### Summary:

1. When Active service instances are available, this tab is now selected by default.
2. The size of the section increased(making sure the Instance Count History section is not truncated).
3. The column size of the environment  increased slightly
4. Added test cases also

##### Jira Links:

https://harness.atlassian.net/browse/CDS-37138

##### Screenshots:

1. ActiveServiceInstance tab
image

2. Deployments tab
image

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by Sagar-Rajput7** [comment]
retrigger titlecheck

File: src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstancesHeader.tsx [hunk 1]
```diff
@@ -98,15 +99,17 @@ export const ActiveServiceInstancesHeader: React.FC = () => {
...
-        {pieChartProps.items.map(({ label, formattedValue, value, color }) => (
-          <li style={{ fontWeight: 500, fontSize: 15, color }} key={`${label}_${value}`}>
-            <Text font={{ size: 'xsmall' }} color={Color.GREY_500} key={label}>{`${label} (${
-              formattedValue ? formattedValue : value
-            })`}</Text>
-          </li>
-        ))}
-      </ul>
+      {!isEmpty(pieChartProps.items) ? (
+        <ul>
```
Comment (kapilkumawat86) [inline src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstancesHeader.tsx:103]: Wh yare we adding inline styles, on uicore components its fine, but pure HTML component use classes instead

File: src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstances.module.scss [hunk 2]
```diff
@@ -41,15 +41,17 @@
     }
     .envContainer {
       padding: var(--spacing-small);
-      padding-left: 0px;
+      display: flex;
+      background-color: var(--lime-400) !important;
+      width: 84px;
+      height: 24px;
```
Comment (kapilkumawat86) [inline src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstances.module.scss:47]: I think for this we have variables, see figma and also compare with uicore vars

File: src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstancesHeader.tsx (hunk 1 above)
Comment (Sagar-Rajput7) [reply src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstancesHeader.tsx:103]: Added classes 👍

File: src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstances.module.scss (hunk 2 above)
Comment (Sagar-Rajput7) [reply src/modules/75-cd/components/ServiceDetails/ActiveServiceInstances/ActiveServiceInstances.module.scss]: fixed and used variables at most places expect 2-3 where we require specific and that is not present in uicore 👍

**Review by kapilkumawat86 — APPROVED** [review]

**Review by PratyushGar — APPROVED** [review]

**Pull request merged**